sklearn/linear_model/coordinate_descent.py

```diff
@@ -1086,7 +1086,7 @@ def fit(self, X, y):
 
         y : array-like, shape 
               - (n_samples,), valid for `ElasticNetCV` and `LassoCV`.
-              - (n_samples, n_targets), valid for `MultiTaskElasticNetCV` and `MultitaskLassoCV`.
+              - (n_samples, n_tasks), valid for `MultiTaskElasticNetCV` and `MultitaskLassoCV`.
```

Comment:
Is n_tasks used elsewhere in scikit-learn?

Reply:
Yes, it is the term used in MultiTaskElasticNet to refer to the extra dim of y, intercept_ and coef_